src/generate-from-template.js

```diff
@@ -52,6 +54,7 @@ const generateFromTemplate = async ({ templateName }) => {
     templateValues,
     generatorOptions = {},
     dynamicFileNames = [],
+    dynamicFolderNames = [],
```

Comment:
for consistency with existing api please use directory instead of folder
```suggestion
    dynamicDirectoryNames = [],
```